joomla/joomla-cms
# [4.0] Clean up the editors

Pull Request for Issue # .

### Summary of Changes
- Fix various bugs,
- Remove deprecated code (php side)
- Remove deprecated code (js side)


### Testing Instructions
Apply patch, select none as your editor check that all the buttons work correctly and the content is saved on clicking the save button.

Repeat for Codemirror and TinyMCE


### Expected result



### Actual result



### Documentation Changes Required
nope!

@wilsonge @mbabker please review the PHP part
